src/main/java/com/betanunes/vinylshelf/controller/VinylController.java

```diff
@@ -0,0 +1,22 @@
+package com.betanunes.vinylshelf.controller;
+
+import com.betanunes.vinylshelf.client.DiscogsClient;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class VinylController {
+    private static final int PAGE_SIZE = 20;
+    private final DiscogsClient discogsClient;
+    public VinylController(DiscogsClient discogsClient) {
+        this.discogsClient = discogsClient;
+    }
+
+    @SuppressWarnings("checkstyle:MagicNumber")
+    @GetMapping("/search")
+    public ObjectNode search(@RequestParam("q") String query)  {
+        return discogsClient.search(query, "master", 1, PAGE_SIZE);
```

Comment:
`"page"` must be a query parameter of the request.

```suggestion
    public ObjectNode search(@RequestParam("q") String query,
                             @RequestParam("page", required = false, defaultValue = "1") String page
    )  {
        return discogsClient.search(query, "master", page, PAGE_SIZE);
```